canonical-web-and-design/maas-ui
# List ssh keys

Done:
- Fetch sshkeys.
- Display the sshkeys in a table. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1506
- Add a component for toggling expanding rows and dedupe existing buttons. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/190.

QA:
- In your MAAS add some ssh keys (/MAAS/account/prefs/).
- Visit /account/prefs/ssh-keys.
- Check that you get a list of ssh keys.
